### PR TITLE
Fix font tests on Linux/Windows

### PR DESCRIPTION
**Pull request by igordmn**
- Fix fonts for Windows (it had different metric for Windows)
- Fix font tests for Linux
- Remove desktop/ParagraphTest (it makes a huge screenshot, and can't be tested reliable)

### CODE REVIEW

File: compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt [hunk 1]
```diff
@@ -414,25 +414,17 @@ class DesktopParagraphTest {
 
     @Test
     fun getBoundingBox_multicodepoints() {
-        assumeTrue(isLinux)
```
Comment (igordmn) [inline compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt:417]: Fails on local Linux, but doesn't fail on CI

File: compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt [hunk 2]
```diff
@@ -619,40 +611,43 @@ class DesktopParagraphTest {
     @Test
     fun applies_baseline_shift_to_spans() {
```
Comment (igordmn) [inline compose/ui/ui-text/src/desktopTest/kotlin/androidx/compose/ui/text/DesktopParagraphTest.kt:612]: Fails on CI: https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_PublishFast_1ComposeDesktopTest/4286199?buildTab=tests&status=muted&expandedTest=build%3A%28id%3A4286199%29%2Cid%3A2000001122

File: compose/desktop/desktop/src/jvmTest/kotlin/androidx/compose/desktop/ParagraphTest.kt [hunk 3]
```diff
@@ -1,124 +0,0 @@
...
-import org.junit.Rule
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.JUnit4
-
-@RunWith(JUnit4::class)
-@OptIn(InternalTestApi::class)
-class ParagraphTest {
-    @get:Rule
-    val screenshotRule = DesktopScreenshotTestRule("compose/ui/ui-desktop/paragraph")
```
Comment (MatkovIvan) [inline compose/desktop/desktop/src/jvmTest/kotlin/androidx/compose/desktop/ParagraphTest.kt:52]: Should we also remove golden images from that path?

Comment (igordmn) [reply compose/desktop/desktop/src/jvmTest/kotlin/androidx/compose/desktop/ParagraphTest.kt:52]: Good catch, removed

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**